docs/TROUBLE.md

```diff
@@ -0,0 +1,175 @@
+
+Start your ground station software and connect to your Tx module by whatever method you have chosen.  You should see or hear the flight mode announced.  If parameter download also completes successfully, you can skip the next two topics
+
+This is a complex topic since there are so many ways to connect a Tx module to a ground station.
+
+- Use the [mLRS LUA script](LUA.md#usage) or [CLI](CLI.md) to check that "Tx Ser Dest" is correct and that "Tx Ser Baudrate" matches your wireless bridge or ground control setting.
+
+- If the data path between your Tx module and ground station includes a serial port, double check your serial port wiring between the Tx module and the ground station or wireless bridge; ensure you have connected serial rx on one board to serial tx on the other board in both directions and ground to ground.
+
+- If your Tx module is the [R9M](FRSKY_R9.md#r9m-tx-module), don't forget to account for the inverted serial port and set dip switch 1 to "on" before powering up.
+
+- If the ground station runs on a mobile device and is connected via USB, an OTG adapter may be required.
+
+- If you want to connect via Bluetooth to the original [MatekSys mLRS Tx Module Kit](MATEKSYS.md), don't forget to enable the HC04 Bluetooth module by moving all three [dip switches](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-2) to the "on" position.
+
+- If the connection is via Bluetooth, the devices need to be paired by your operating system first.
+
+- If the connection is via UDP or TCP over WiFi, connect to WiFi access point before starting the ground station.
+
+## 6. Confirm MAVLink message flow from receiver to flight controller
+
+Use the mLRS LUA script or another configuration method to temporarily set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to "rc override".  If you use the mLRS LUA script there is no need to save; Just press the RTN button after selecting "rc override".
+
+Connect a ground station via USB directly to your flight controller and view the MAVLink messages.
+With QGroundControl, use "Analyze Tools" -> "MAVLink Inspector" and select System 255 in the upper right.
+With Mission Planner, Press Ctrl-F, select "MAVLink Inspector" and expand "Vehicle 255" -> "Comp 68" -> "RC\_CHANNELS\_OVERRIDE".
+
+You should see the RC\_CHANNELS\_OVERRIDE message is being received at the same rate as your configured mLRS mode.  Note that this also confirms your Tx module is receiving RC channel data from your radio and sending it to the Rx.
+
+- If you don't see these messages, double check your wiring to ensure that the correct serial tx output on the receiver module is connected to the correct rx input on your flight controller.
+
+## 7. Confirm MAVLink message flow from ground station to flight controller
+
+Using your ground station connected to your Tx module, verify that you can change flight modes (Using the ground station, not the radio) and successfully download parameters.
```

Comment:
Perhaps 'verify that you can change flight modes using the ground station and that you can successfully download parameters.'

Reply:
I think I need to say "not with the radio" somewhere lest they think I mean change the flight mode with the radio switch and verify the change using the groundstation.  Do you have a suggestion that makes that clear?

Reply:
My quick thought is that ensuring parameters can be downloaded would be a successful pass of this test (as GCS needs to send the request to FC for this) so perhaps it makes sense to just remove mentions of flight mode?

Reply:
Works for me